frontend/src/app/main/data/team.cljs

```diff
@@ -282,7 +282,11 @@
   [team]
   (ptk/reify ::team-created
     IDeref
-    (-deref [_] team)))
+    (-deref [_] team)
+
+    ptk/UpdateEvent
+    (update [_ state]
+      (dissoc state :files))))
```

Comment:
This is not a good fix, because it removes all the pre "cached" files

Reply:
You are right <3